### PR TITLE
Decrypt hashes

### LINKED ISSUE #4: Translate hashes to objects
- Do what `git cat-file -p` does

### LINKED ISSUE #7: Find a library for git analysis (git object decompression)
Look into pydriller

### PR DESCRIPTION
**Pull request by tjomson**
Co-Authored-By: emiljapelt <55390848+emiljapelt@users.noreply.github.com>
closes #4 

Also resolves #7 (we are using zlib)

### CODE REVIEW

File: parser/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "parser",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (joglr) [inline parser/package.json:7]: Could you add a build script?

**Review by joglr — CHANGES_REQUESTED** [review]

File: parser/package.json (hunk 1 above)
Comment (tjomson) [reply parser/package.json]: Added both 😎

File: parser/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            const fileContent = await parseFile(dir + "/" + fileName);
+            console.log(fileContent);
+            gitObjects.set(entry + fileName, fileContent);
+        }));
+        
+    });
+    console.log(gitObjects);
+}
+
+parseGitObjects('../../bprd');
```
Comment (joglr) [inline parser/src/index.ts:32]: Could this take a CLI arg for the requested path?

**Review by joglr — APPROVED** [review]

**Review by emiljapelt — APPROVED** [review]
yep, I say that something worked

File: parser/src/index.ts (hunk 2 above)
Comment (tjomson) [reply parser/src/index.ts]: Good idea 🚀🚀

**Pull request merged**